.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
```

Comment:
for posterity, this is removed by/in husky v9.1.1 : https://github.com/typicode/husky/issues/1472#issuecomment-2236089807